test/libsolidity/semanticTests/externalContracts/_prbmath/PRBMathSD59x18.sol

```diff
@@ -0,0 +1,578 @@
+// SPDX-License-Identifier: WTFPL
+pragma solidity >=0.8.0;
+
+import "./PRBMathCommon.sol";
```

Comment:
@cameel Why isn't this working?

```
semanticTests/externalContracts/prbmath.sol: Warning: This is a pre-release compiler version, please do not use it in production.

Error: Source "_prbmath/PRBMathCommon.sol" not found: File not supplied initially.
 --> _prbmath/PRBMathSD59x18.sol:5:1:
  |
5 | import "./PRBMathCommon.sol";
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Exception during test: /Users/alex/Projects/solidity/libsolidity/interface/CompilerStack.cpp(703): Throw in function const std::__1::string solidity::frontend::CompilerStack::lastContractName(const optional<std::__1::string> &) const
Dynamic exception type: boost::wrapexcept<solidity::langutil::CompilerError>
std::exception::what: Parsing was not successful.
[solidity::util::tag_comment*] = Parsing was not successful.
```

Reply:
Do I need to specify each used ExternalSource? :(

Reply:
Yeah, you do because there is no `FileReader` in tests. So you need to populate the VFS with sources. Before external tests there was no way to do that at all outside of multisource tests, now you can - with `ExternalSource`.

It's not like we can't add a `FileReader` there though. Now that the PR is merged I could try to do it if you think it's needed. We decided to keep it out of the original PR for simplicity. The original solution of manually parsing the file, extracting imports, guessing source unit names and then letting `CompilerStack` parse it again was not the way to go IMO.

Reply:
I'm not convinced this is what we wanted to have in the first place, but I guess we can live with it.

Reply:
Do we have any spec on what we wanted? Because that's not very clear to me. I only know that what I saw originally in the PR was overcomplicating the implementation so I suggested simplifying it. I think it's not hard to now extend this simplified base to make it more convenient - you just have to state your requirements :)